src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,30 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String name;
+    private String color;
+    private String mover;
+    private int wide;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. You should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Consider marking these fields as private.